templates/test/overview.html.ep

```diff
@@ -11,10 +11,10 @@
     <div id="summary" class="panel <%= ($aggregated->{failed} + $aggregated->{incomplete}) ? 'panel-danger' : 'panel-success' %>">
         <div class="panel-heading">
             Overall Summary of
-            % if ($group) {
-                <b><%= link_to $group->name => url_for('group_overview', groupid => $group->id) %></b>
-            % } else
-            % {
+            % if (@$groups) {
+                <b><%= b join(', ', map { link_to $_->name => url_for('group_overview', groupid => $_->id) } @$groups) %></b>
```

Comment:
That "Summary of $distri $version $build" would be an intermediate level, I can try this.

> Mixing flavors between groups sounds suboptimal

Just allowing the query to combine any combination does not necessarily make it very useful. My intention was to first allow *everything* with a query which would also work in a json representation and then we can think about how to generate these queries, e.g. adding a link to a build from parent job groups directly.